packages/ui-components/src/StringArrayInput.tsx

```diff
@@ -0,0 +1,293 @@
+/*
+ * Copyright 2018-2021 Elyra Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { editIcon } from '@jupyterlab/ui-components';
+import {
+  Button,
+  ButtonGroup,
+  IconButton,
+  InputLabel,
+  List,
+  ListItem,
+  ListItemText,
+  TextField
+} from '@material-ui/core';
+import { FormHelperText } from '@material-ui/core';
+import produce from 'immer';
+import { useCallback, useEffect, useRef, useState } from 'react';
+
+import React from 'react';
```

Comment:
I think we've been using `React.useXXX` in the main elyra repo
```suggestion
import * as React from 'react';
```